src/lib/registration/ui/base_system_registration_dialog.rb

```diff
@@ -395,49 +394,54 @@ def medium_warning_text
         Y2Packager::MediumType.online? ? online_skipping_text : default_skipping_text
       end
 
-      def default_skipping_text
-        # TRANSLATORS:
-        # Popup question (1/1): confirm skipping the registration
-        warning = _("You are skipping registration.\n"\
-          "Please configure access to packages medium in the next step.\n"\
-          "\n"\
-          "Without registration update-channels will not be configured.\n"\
-          "This will disable the updates and security fixes.")
-
-        # TRANSLATORS:
-        # Popup question (2/2): confirm skipping the registration
-        # %{media_name} is the media name (e.g. SLE-15-Packages),
-        # %{download_url} is an URL link (e.g. https://download.suse.com)
-        if !media_name.empty? && # cannot be nil
-            !download_url.empty? # cannot be nil
-          warning += "\n\n" +
-            _("If you do not register AND do not use the %{media_name}\n"\
-              "medium from %{download_url}\n"\
-              "then only an installation system is installed.\n"\
-              "\n"\
-              "The installation system is not intended to be used as\n"\
-              "an operational system. The installation system can only\n"\
-              "be used to install a working system.") %
-            { media_name: media_name, download_url: download_url }
-        end
+      def medium_warning_headline
+        Y2Packager::MediumType.online? ? online_skipping_headline : default_skipping_headline
+      end
 
-        warning
+      def online_skipping_headline
+        # TRANSLATORS: popup header
+        _("Registration Cannot be Skipped")
```

Comment:
:thinking: be / Be :thinking:

Reply:
Ah, I'm always confused how to write the title style capitalization correctly. It seems there is even an online tool for that https://capitalizemytitle.com/

Reply:
Cool!, Didn't know about it ;)